pkg/ddl/schema_version.go

```diff
@@ -266,17 +266,12 @@ func SetSchemaDiffForCreateTable(diff *model.SchemaDiff, job *model.Job) error {
 
 // SetSchemaDiffForRecoverSchema set SchemaDiff for ActionRecoverSchema.
 func SetSchemaDiffForRecoverSchema(diff *model.SchemaDiff, job *model.Job) error {
-	var (
-		recoverSchemaInfo      *RecoverSchemaInfo
-		recoverSchemaCheckFlag int64
-	)
-	err := job.DecodeArgs(&recoverSchemaInfo, &recoverSchemaCheckFlag)
+	args, err := model.GetRecoverArgs(job)
 	if err != nil {
 		return errors.Trace(err)
 	}
 	// Reserved recoverSchemaCheckFlag value for gc work judgment.
-	job.Args[checkFlagIndexInJobArgs] = recoverSchemaCheckFlag
```

Comment:
Reserved recoverSchemaCheckFlag ?

Reply:
I don't know why it decode `recoverSchemaCheckFlag` and set it again here. I'll remove the comment here.

Reply:
might because it became a pointer after DecodeArgs